backend/nodes/image_nodes.py

```diff
@@ -69,11 +69,21 @@ def get_extra_data(self) -> Dict:
 
         import base64
 
+        # resize the image, so the preview loads faster and doesn't lag the UI
+        # 512 was chosen as the target because a 512x512 RGBA 8bit PNG is at most 1MB in size
+        target_size = 512
+        max_size = target_size * 1.2
```

Comment:
What exactly is the point of this * 1.2?

Reply:
If the image is `target_size + 1` (or some other small integer), then the downscaled image will be a complete mess, I think. Maybe it's not a huge issue, but I wanted to be sure.

Also, are the interpolation methods that don't have this problem?

Reply:
Area is the best, at least in opencv. In what way does the result become a mess?

Reply:
It will become blurry. Here's an image with a width of 513 without that check:

![image](https://user-images.githubusercontent.com/20878432/166570901-a91e1e31-24f8-4fd3-9468-8f27680a088a.png)

and with the check:

![image](https://user-images.githubusercontent.com/20878432/166571073-e5d243f2-047a-4cae-a2c6-0f5ebdc57e86.png)

It's not *very* blurry, but it's definitely better with the check.

Reply:
That being said, most of the image I upscale are have powers of two as their size, so I personally wouldn't have a problem with the check being removed.

Reply:
Oh I understand now, you only wanna resize it if it's more than just a bit larger than the target size. I see.